sdk/typespec/typespec_derive/src/lib.rs

```diff
@@ -0,0 +1,82 @@
+use syn::{parse::ParseStream, parse_macro_input, spanned::Spanned, DeriveInput, Error, LitStr};
```

Comment:
Copyright headers on all sources per corporate policy. `.editorconfig` has something, but seems we don't enforce it (it's non-standard but common `.editorconfig`).

Reply:
Hmm... wonder if there's a VS Code extension that could do that automatically 🤔